### DISCUSSION TITLE
Consider adopting WebC for docs

### DISCUSSION THREAD

**Discussion by bennypowers** [RedHat-UX/red-hat-design-system#804] [Ideas]
[Bryce Wray makes a good argument for adopting WebC over nunjucks for 11ty templating](https://www.brycewray.com/posts/2023/03/time-move-on-nunjucks/). In particular, we could benefit from [WebC's Declarative Shadow DOM support](https://www.11ty.dev/docs/languages/webc/#compatible-with-standards)

**Comment by zeroedin** [comment]
+1

**Comment by bennypowers** [comment]
cc @nikkimk see https://www.11ty.dev/docs/languages/webc/

**Reply by nikkimk** [reply]
I'm in!
